### DISCUSSION TITLE
How to delete the key "output" in Wflow.Config Object in julia?

### DISCUSSION THREAD

**Discussion by watercore2001** [Deltares/Wflow.jl#445] [Q&A]
Sometimes I don't want wflow to output nc files. One solution is to remove the output-related field in the toml configuration file. However, I want to use a unified configuration file, so I want to control this field in the Julia code.

So How to delete the "Output" key in Wflow.Config in julia？

below is my trying code:
```
# get a wflow Config Object
config = Wflow.Config(args["config_path"])
println(config)
# try to delete, error
delete!(config, "output")

# output: 
Wflow.Config(Dict{String, Any}.......)
ERROR: LoadError: MethodError: no method matching delete!(::Wflow.Config, ::String)
```

**Comment by JoostBuitink** [comment]
You can use `pop!(config, key)` to delete keys in the config object, which should do exactly what you want to achieve in this workflow.
